src/main/scala/strawman/collection/mutable/Iterator.scala

```diff
@@ -4,9 +4,10 @@ import scala.{Boolean, Int, Unit, Nothing, NoSuchElementException}
 import strawman.collection.{IndexedView, IterableOnce}
 
 /** A core Iterator class */
-trait Iterator[+A] { self =>
+trait Iterator[+A] extends IterableOnce[A] { self =>
```

Comment:
But that's why it says `IterableOnce`! It means you can take one `iterator()`, but not several ones. Note that `TraversableOnce` was not part of 2.8 collections. It got added later by @paulp, because we experienced a lot of pain regarding the divergence of `Iterator` and `Iterable`.